gce-production-1/instance-counts.auto.tfvars

```diff
@@ -1,5 +1,5 @@
 {
-  "worker_managed_instance_count_com": 27,
-  "worker_managed_instance_count_org": 33,
+  "worker_managed_instance_count_com": 18,
```

Comment:
not a blocker for merge, but more a curiosity on my part, how do we calculate the managed instance count (or pool size math also) for GCE?